### PR TITLE
10476-Initial mpHealth-2.2 implementation

### LINKED ISSUE #10476: Initial feature delivery for mpHealth-2.2 Implementation
- Create new mpHealth-2.2 public beta feature
- Add the Eclipse MicroProfile Health Check 2.2 libraries to Liberty
- Implement the new functionality for MicroProfile Health Check 2.2 Specification in OpenLiberty.
- Add the TCK project to run the MicroProfile Health Check 2.2 TCK with OpenLiberty, to ensure compatibility.

### PR DESCRIPTION
**Pull request by pgunapal**
Fixes #10476

### CODE REVIEW

**Comment by pgunapal** [comment]
No ID/L2 review required, as the message files are just a copy from mpHealth-2.0 feature files, no new messages were added/updated.

**Comment by pgunapal** [comment]
#build

**Comment by pgunapal** [comment]
#build

File: dev/com.ibm.ws.microprofile.health.2.0/src/com/ibm/ws/microprofile/health20/internal/HealthCheck20ServiceImpl.java [hunk 1]
```diff
@@ -107,7 +107,7 @@ public void performHealthCheck(HttpServletRequest request, HttpServletResponse h
             if (!appTracker.isStarted(appName)) {
                 if (tc.isDebugEnabled())
                     Tr.debug(tc, "In performHealthCheck(): Application : " + appName + " has not started yet.");
-                if (healthCheckProcedure == HealthCheckConstants.HEALTH_CHECK_READY || healthCheckProcedure == HealthCheckConstants.HEALTH_CHECK_ALL) {
+                if (healthCheckProcedure != HealthCheckConstants.HEALTH_CHECK_LIVE) {
```
Comment (fmhwong) [inline dev/com.ibm.ws.microprofile.health.2.0/src/com/ibm/ws/microprofile/health20/internal/HealthCheck20ServiceImpl.java]: Comparing Strings with != and == ?

Comment (pgunapal) [reply dev/com.ibm.ws.microprofile.health.2.0/src/com/ibm/ws/microprofile/health20/internal/HealthCheck20ServiceImpl.java]: Forgot to push that change in, good catch!

**Comment by pgunapal** [comment]
#build

**Comment by pgunapal** [comment]
#build

**Comment by pgunapal** [comment]
#run-libby-bot

**Pull request merged**